lib/publiccloud/instance.pm

```diff
@@ -388,7 +388,7 @@ sub softreboot
         ssh_interactive_leave();
     }
 
-    $self->run_ssh_command(cmd => 'sudo shutdown -r +1');
+    $self->ssh_script_run(cmd => 'sudo shutdown -r +1');
```

Comment:
just to keep things clear - this is unrelated , right ? ( it is fine just for general understanding )

Reply:
This is required so that we use the modified `_prepare_ssh_cmd` subroutine